api/src/main/java/com/spotify/ffwd/output/FlushingPluginSink.java

```diff
@@ -402,17 +399,23 @@ Batch newBatch() {
     static class Batch {
```

Comment:
is it a good name and location for a class that represents collection of batches?

Reply:
It's purely internal to this class, so name doesn't matter (much).